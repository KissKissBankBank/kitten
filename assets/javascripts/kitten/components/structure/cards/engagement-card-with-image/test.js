import React from 'react'
import renderer from 'react-test-renderer'
import { EngagementCardWithImage } from './index'

describe('<EngagementCardWithImage />', () => {
  let component

  describe('by default', () => {
    beforeEach(() => {
      component = renderer
        .create(
          <EngagementCardWithImage
            backgroundImage="kitten.jpg"
            href="#"
            title="Hello word"
            imgProps={{ srcSet: 'kitten.jpg 1x, kitten@2x.jpg 2x' }}
            textProps={{ 'aria-label': 'test label' }}
          />,
        )
        .toJSON()
    })

    it('matches with snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
