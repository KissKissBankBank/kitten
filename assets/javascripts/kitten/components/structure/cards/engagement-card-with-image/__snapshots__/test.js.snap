// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EngagementCardWithImage /> by default matches with snapshot 1`] = `
<a
  className="sc-bdVaJa iCkDfz"
  href="#"
  title="Hello word"
>
  <div
    className="sc-bwzfXH chwYRj"
  >
    <img
      alt=""
      className="sc-htpNat bvrdCN"
      src="kitten.jpg"
      srcSet="kitten.jpg 1x, kitten@2x.jpg 2x"
    />
  </div>
  <span
    aria-label="test label"
    className="k-u-color-font1 k-u-decoration-none k-u-line-height-normal k-u-size-micro k-u-weight-regular sc-bxivhb iSbjCA"
  />
</a>
`;
